### ISSUE TITLE
Tiled Sprite object does not repeat some sprites, instead it stretches them.

### ISSUE THREAD

**Issue by Lumios64** [4ian/GDevelop#2761]
## The bug
In short, I discovered a bug by accident by messing around with Gdevelop's editor. This bug makes it so any tiled sprite (Or as far as I can tell) become stretched-out instead of repeating the intended pattern on the sprite.

Based on extensive testing on my end, it affects the editor, the preview and the exported game, so pretending that the bug isn't there is not an option. It affects a specific type of sprite of a certain height and length. 

The bug might affect only some computers for which are caused by the CPU or GPU.

## System details
* Debian 8 running on a Packard Bell laptop with an AMD processor
* 5.0.0-beta110 AppImage

## To Reproduce
I have a project file with the bug already there, so you may take a look at it instead.

1. Create any tiled sprite with the following resolutions: 16x16, 32x32, 64x64 or any other one with a 1:1 ratio. With that said I have only tested the aforementioned resolutions.
2. Import the file into the project.
3. Create a tiled sprite with the file.
4. Finally, increase its size via the editor and the sprite will appear stretched.

* Please include a link to a game if possible!
I have zipped a project made to showcase the issue.
[ISSUE.zip](https://github.com/4ian/GDevelop/files/6717759/ISSUE.zip)

* If applicable, add screenshots to help explain your problem.
Sure.
![Untitled](https://user-images.githubusercontent.com/85809526/123460663-4152b380-d5e8-11eb-86fc-502e96d49a96.png)
I would have used a video to better illustrate the problem, but it would take me days to install any software that's not from the app manager on my system due to how cumbersome Linux can get, and I have to make sure it actually works too. If that is demanded or asked I can always do that though.

## Workaround
If this bug does affect one of your projects, make sure to give the sprite any ratio that is not 1:1, it can be 1:2, 1:3, 2:3, 3:4, etc and it will work as intended again. That is how I figured this problem.

## Final comment
The stretching effect could be added into Gdevelop as an intended feature. I could see it being useful for GUIs, however, as of now, it is rather annoying.

**Comment by willianholtz** [comment]
I downloaded your problem, and I couldn't reproduce what you did in the image! 

Resize:
![Screenshot_20210625_161337](https://user-images.githubusercontent.com/23488207/123474433-528fc600-d5d0-11eb-8ea8-ff341489d16a.png)


Preview:
![Screenshot_20210625_161611](https://user-images.githubusercontent.com/23488207/123474687-b5815d00-d5d0-11eb-8438-f16e9d8c93d2.png)


And I use linux (Arch Linux)

**Comment by Lumios64** [comment]
I cannot reproduce the issue on Windows. 

So now I'm wondering whether this is a problem with how Gdevelop handles the graphics which would be directly an issue with the processor itself. My Windows has an Intel processor while Linux uses AMD, what's yours @willianholtz? 

I'll update the OP.

**Comment by willianholtz** [comment]
It must be something related to your Debian's AMD driver.

I use a radeon on my Arch linux and it's perfect! I never had a problem. Now the fact of using it on Windows, by testing on Windows 7, the same thing, it doesn't show any problems.

**Comment by Lumios64** [comment]
Hey, I also use Windows 7. I thought most people ditched it for Win 10, but I guess not.
Anyway, The Linux's CPU is an AMD E1-1200 and the GPU a Radeon HD 7310. From what I read this is a common spec for laptops.

**Comment by Silver-Streak** [comment]
I can't reproduce this either, but I'm also updating the title of this issue as this is unrelated to Tilemaps. (Tiled Sprite objects have no relation to the Tilemap object).

**Comment by 4ian** [comment]
> make sure to give the sprite any ratio that is not 1:1, it can be 1:2, 1:3, 2:3, 3:4, etc and it will work as intended again. That is how I figured this problem.

Definitely sounds like a GPU problem, especially if it can't be reproduced consistently on other computer/phones.

**Comment by Bouh** [comment]
I add a flag for get more user tests and I close because it seem related to the graphic card / drivers.

**Comment by Lumios64** [comment]
As of version 5.0.134, the issue has been solved. 

The image below showcases the bug stated on the ticket, but it is a much bigger image than the one included in that ticket. It was taken sometime after the ticket was tagged and closed.
![Problem](https://user-images.githubusercontent.com/85809526/170274483-4970b02a-5a58-481b-a225-3d83ef862d2d.png)

The image below showcases the same set-up on a different and newer project. Notice that the bug does not occur contrary to the screenshot above.

I included a second screenshot with the current build of GDevelop on it.
![ProblemSolved](https://user-images.githubusercontent.com/85809526/170274501-29dda933-3154-45e4-90f2-bdbcab32eaaf.png)
![CurrentVersion](https://user-images.githubusercontent.com/85809526/170274529-9d14a0ff-298d-4867-9963-57e03a118057.png)

This was tested on the same system which includes the same graphical specs where I had first encountered the issue.
